### DISCUSSION TITLE
Gradle in root and sudirectory not updated

### DISCUSSION THREAD

**Discussion by Hungon** [renovatebot/renovate#11477] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Describe the bug

Renovate does not create PR even though `build.gradle` exists at the root of the project.

It seems Renovate does not detect Gradle file properly.
Here is debug log:
```
INFO: Dependency extraction complete
{
  "baseBranch": "main",
  "stats": {
    "managers": {
      "bundler": {
        "fileCount": 1,
        "depCount": 2
      },
      "gradle-wrapper": {
        "fileCount": 1,
        "depCount": 1
      },
      "ruby-version": {
        "fileCount": 1,
        "depCount": 1
      }
    },
    "total": {
      "fileCount": 3,
      "depCount": 4
    }
  }
}
```

There should be `gradle`  like the below:
```
"gradle": {
        "fileCount": 5,
        "depCount": 320
      }
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment, marked as answer]
Please provide your config, it seems you've enabled only specific managers or diusabled some managers ...

**Reply by Hungon** [reply]
Sorry for the late reply.
We're configuring with the preset base config.
After posting the issue, now it's working normally.
Thank you for your help.
